### ISSUE TITLE
Drop ships have exteriors, and can be breached with welders/bomb/etc.

### ISSUE THREAD

**Issue by Admiral-Gigaton** [BeeStation/NSV13#2500]
Issue reported from  Round ID: 6541 (NSV 13 | Official | MRP | 98% Free Lag | Spaceship Combat Server)

Reporting client version: 514.1589

title. you can weld the walls of a dropship, leading to space, possibly resulting in ultimate tot hideout inside a dropship space tile. solution: disable teleporting in the dropship Z level, and make dropship walls into invincible CC walls.

**Comment by Brillcrafter** [comment]
I disagree with the teleporting being blocked,   due to the nature of the drop ship it doesn't really make sense to stop teleporting. However you could make the area of space outside the drop ship smaller so anyone with any of the goggles can see outside. I do agree with making the inside walls indestructible and I would like that change applied to the mining shuttles as will
